slide_analysis/utils/tile_stream_class.py

```diff
@@ -0,0 +1,17 @@
+class TileStream:
+    def __init__(self, func, length):
```

Comment:
I think tileStream should be coonected to splitting service closely
In current implementation tileStream is no more than some class that applies function with arguments 0...n and returns value
basicly 
```python
for i in range(n):
     func(i)
```
That can be done without service
We need some connection between tile stream and image
So we can determine its file_name and any other image property
for example (size method for splitting service should be implemented)
```python
class TileStream:
    def __init__(self, splitting_service):
        self.iteration = 0
        self.splitting_service = splitting_service

    def next(self):
        res = self.splitting_service.cut_tile(self.iteration)
        self.iteration += 1
        return res

    def has_next(self):
        return self.iteration < self.splitting_service.size()

    def for_each(self, func):
        while self.has_next():
            func(self.next())

```